src/core/configuration.c

```diff
@@ -52,12 +52,6 @@ MsQuicConfigurationOpen(
         goto Error;
     }
 
-    if (Settings != NULL &&
```

Comment:
Why no size check any more?

Reply:
I want to forward all size checks to QuicSettingsSetSettings. That way they only exist in 1 spot. Need to finish up the work doing that.